pyof/foundation/base.py

```diff
@@ -342,6 +342,52 @@ def __new__(mcs, name, bases, classdict):
 
         return type.__new__(mcs, name, bases, classdict)
 
+    @staticmethod
+    def _get_module_version(fullname):
+        ver_module_re = re.compile(r'(pyof\.)(v0x\d+)(\..*)')
+        version = None
+        # module = None
+        requested = ver_module_re.match(fullname)
+        if requested:
+            version = requested.group(2)
+            # module = requested.group(3)
+            return version
+        return None
+
+    @staticmethod
+    def _replace_attribute_version(module, version):
+        module_version = MetaStruct._get_module_version(module)
+        if not module_version:
+            return module
```

Comment:
The linter should complain about missing docstring. Explain this in a docstring would tackle both problems.